pages/faq/index.html

```diff
@@ -144,4 +144,34 @@ <h3 class="title title-two contentbox-title">Why are the badge-numbers of the pl
                 lists all FMUs or Cross-Check Results submitted.</p>
         </p>
     </article>
+	<article class="contentbox">
+        <h3 class="title title-two contentbox-title">How are the FMI version numbers defined and what does backwards compatibility of a minor version mean?</h3>
+        <p class="contentbox-content">
+            <p>According to the FMI development process (<a href="https://svn.fmi-standard.org/fmi/branches/public/docs/DevProcess/FMI_DevelopmentProcess_1.0.pdf">link</a>), the version number of an FMI release is defined in the following way:
+			</p>
+			<ul>
+                <li>Version number: MajorVersion.MinorVersion.MaintnanceVersion</li>
+                <li>Changes introduced by Major versions don't have to be backward nor forward compatible.</li>
+				<li>Minor versions within the same Major version have to be backward compatible.</li>
+				<li>Maintenance versions must not introduce new features and have to be forward and backward compatible.</li>
+				<li>All versions will have semantic version numbers.
```

Comment:
Strictly speaking yes. But do you see a problem in calling it colloquially "FMI 3.0" just as we talk about "FMI 1.0", "FMI 2.0", and only use the third level only for bugfix releases?
Bugfix release is forward and backwards compatible and contain the same features and have a version string "FMI 3.0".

Reply:
It might lead to problems when using semver libraries in tool implementations that expect 3 digits and might throw an error. Some do automatic conversions others have to implement that. But it would be much cleaner to call FMI 3.0.0 also 3.0.0 and not 3.0 which then needs to automagically be converted to a 3.0.0 representation to satisfy compatibility logic. So when you say all versions will have semantic version numbers then this should really be the case. 
Just my 2 cents.

Reply:
When we talk about a feature set (as we do in in the prelimary features list), I think it is ok to talk about FMI 3.0.  A release shall have three number (i.e. 3.0.0). A different topic is, if the version string in the modeldescription.xml of of 3.0.x releases shall be "FMI 3.0" or "FMI 3.0.x". I created a trac issue for the discussion of this question: https://trac.fmi-standard.org/ticket/433#ticket